src/Model/ClearableManagerInterface.php

```diff
@@ -19,6 +19,8 @@
 interface ClearableManagerInterface
 {
     /**
+     * NEXT_MAJOR: Remove $objectName parameter.
```

Comment:
```suggestion
     * NEXT_MAJOR: Remove $objectName argument.
```

Reply:
I think [`parameter`](https://developer.mozilla.org/en-US/docs/Glossary/Parameter) here is fine since it's in the method definition.

Reply:
Thank you for the reference. I think that's a good point.